app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -116,7 +116,9 @@ class AppVersionActivityTest {
 
   @get:Rule
   val activityTestRule: ActivityTestRule<AppVersionActivity> = ActivityTestRule(
-    AppVersionActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+    AppVersionActivity::class.java, /* initialTouchMode= */
+    true, /* launchActivity= */
+    false
```

Comment:
These changes can be un-done.

`AppVersionActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false`